bigquery/google/cloud/bigquery/dataset.py

```diff
@@ -221,8 +221,8 @@ def created(self):
             return _datetime_from_microseconds(1000.0 * creation_time)
 
     @property
-    def dataset_id(self):
-        """ID for the dataset resource.
+    def full_dataset_id(self):
```

Comment:
It is a little whiplash-y to me that what _used_ to be spelled `dataset_id` is now spelled `full_dataset_id`, and what _used_ to be spelled `name` is now spelled `dataset_id`.

Not opposing outright, but is there a just-as-good terminology that would might be a bit easier on folks?

If the answer is no, then charge ahead. :-)

Reply:
Follow-up: I _definitely_ do like that `dataset_id` matches the server name, though, which is (I assume) why it is being used here. Okay, I think I am tracking why this decision is made.